packages/filesystem/src/browser/file-dialog/file-dialog.ts

```diff
@@ -116,13 +118,15 @@ export abstract class FileDialog<T> extends AbstractDialog<T> {
     protected readonly back: HTMLSpanElement;
     protected readonly forward: HTMLSpanElement;
     protected readonly home: HTMLSpanElement;
+    protected readonly up: HTMLSpanElement;
     protected readonly locationListRenderer: LocationListRenderer;
     protected readonly treeFiltersRenderer: FileDialogTreeFiltersRenderer | undefined;
     protected readonly treePanel: Panel;
 
     constructor(
         @inject(FileDialogProps) readonly props: FileDialogProps,
-        @inject(FileDialogWidget) readonly widget: FileDialogWidget
+        @inject(FileDialogWidget) readonly widget: FileDialogWidget,
+        @inject(FileService) readonly fileService?: FileService
```

Comment:
I believe this should be added to the `changelog` as a breaking change since downstream clients who extend the `file-dialog` will be broken. It is also a breaking change due to: https://github.com/eclipse-theia/theia/wiki/Coding-Guidelines#property-injection.